TestAssets/TestProjects/ResourceLookup/project.json

```diff
@@ -0,0 +1,386 @@
+{
+
+  "buildOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {},
+  "frameworks": {
+    "netcoreapp2.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": {
+          "type": "platform",
+          "version": "2.0.0-preview2-*"
+        },
+        "System.Spatial": "*",
```

Comment:
Instead of a *, can you grab the latest version from NuGet.org and put it in here?  That way our builds and test runs are consistent.